packages/build-utils/test/unit.convert-runtime-to-plugin.test.ts

```diff
@@ -114,35 +114,35 @@ describe('convert-runtime-to-plugin', () => {
       files: [
         {
           input: `../../../inputs/api-routes-python/api/db/[id].py`,
-          output: 'api/db/[id].py',
+          output: '.output/server/pages/api/api/db/[id].py',
         },
         {
           input: `../../../inputs/api-routes-python/api/index.py`,
-          output: 'api/index.py',
+          output: '.output/server/pages/api/api/index.py',
         },
         {
           input: `../../../inputs/api-routes-python/api/project/[aid]/[bid]/index.py`,
-          output: 'api/project/[aid]/[bid]/index.py',
+          output: '.output/server/pages/api/api/project/[aid]/[bid]/index.py',
         },
         {
           input: `../../../inputs/api-routes-python/api/users/get.py`,
-          output: 'api/users/get.py',
+          output: '.output/server/pages/api/api/users/get.py',
```

Comment:
Something seems wrong here. Did you intend to have `api` nested in `api` like this?

Reply:
Yes! All the dependencies are placed next to the launcher.